tests/wdio.conf.js

```diff
@@ -0,0 +1,156 @@
+exports.config = {
```

Comment:
Can we trim this file down to just the bits we need (leave defaults as defaults etc.)? If we've got lots of comments, they're very likely to be ignored and unmaintained.